sdk/formrecognizer/ai-form-recognizer/src/constants.ts

```diff
@@ -1,11 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * Defines known cloud audiences for Form Recognizer.
+ */
+export enum KnownFormRecognizerAudience {
+  /** Azure China */
+  AzureResourceManagerChina = "https://cognitiveservices.azure.cn/.default",
```

Comment:
There is nothing special about these links relating to `FormRecognizer`, should it be `KnownCognitiveServicesAudience` and declared at some common place?

Reply:
That would make sense if we had a common package for cognitive services which we don't...

Reply:
Yes, sad.

Reply:
We can duplicate the code and use better name for now.